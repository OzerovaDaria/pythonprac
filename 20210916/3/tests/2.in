[9, 12, 7, 34, 6, 90]
