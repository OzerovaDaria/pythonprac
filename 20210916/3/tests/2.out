[9, 12, 7, 34, 6, 90, 98, 58, 3, 91, 10, 87, 39, 41, 90]
